teaspoons-cli/teaspoons-client.gradle

```diff
@@ -0,0 +1,18 @@
+tasks.openApiGenerate {
+    generatorName.set("python")
+    inputSpec.set("$rootDir/common/openapi.yml")
+    outputDir.set("$rootDir/teaspoons-cli/build")
+    configOptions.put("projectName", "teaspoons-client")
+    configOptions.put("packageName", "teaspoons_client")
+    configOptions.put("packageVersion", "0.0.0")
+    configOptions.put("httpUserAgent", "teaspoons-client/0.0.0/python")
+}
+
+// because we're building this Python client in a Java project, we add this so that the Python gets generated when we build the project
+compileJava.dependsOn tasks.openApiGenerate
+
+task customCleanUp(type:Delete) {
+    delete "${rootDir}/teaspoons-cli/build/"
+}
+
+tasks.clean.dependsOn(tasks.customCleanUp)
```

Comment:
ooo nice, do our other clients have a similar thing?

Reply:
no, they seem to get cleaned up automagically

Reply:
🪄